docs/app/pages/components/sections/date-time-picker/integrated-date-picker-ng1/integrated-date-picker-ng1.component.html

```diff
@@ -27,6 +27,13 @@
             <td>false</td>
         </tr>
         <tr>
+            <td class="attribute">ng-model-options</td>
```

Comment:
Remove this description as ng-model-options is a built in Angular feature, not part of this directive.